From 69253dd2794d979a56641fc13e14e91ad919ee57 Mon Sep 17 00:00:00 2001
From: Patrick Delaunay <patrick.delaunay@foss.st.com>
Date: Mon, 18 Dec 2023 19:57:23 +0100
Subject: [PATCH 1124/1141] ASoC: dt-bindings: stm32-sai: add 'power-domains'
 property

STM32 SAI may be in a power domain which is the case for the STM32MP25
based boards. Allow a single 'power-domains' entry for STM32 SAI.

Fix dtbs_check warning:
Unevaluated properties are not allowed ('power-domains' was unexpected)

Signed-off-by: Patrick Delaunay <patrick.delaunay@foss.st.com>
Change-Id: Id63ebf29b1a9a681c50d5b7c54423b9c4344a214
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/347854
Tested-by: Eric FOURMONT <eric.fourmont-ext@st.com>
Domain-Review: Eric FOURMONT <eric.fourmont-ext@st.com>
Reviewed-by: Eric FOURMONT <eric.fourmont-ext@st.com>
---
 Documentation/devicetree/bindings/sound/st,stm32-sai.yaml | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/Documentation/devicetree/bindings/sound/st,stm32-sai.yaml b/Documentation/devicetree/bindings/sound/st,stm32-sai.yaml
index 7c0f61928fb3..1a0f692ac958 100644
--- a/Documentation/devicetree/bindings/sound/st,stm32-sai.yaml
+++ b/Documentation/devicetree/bindings/sound/st,stm32-sai.yaml
@@ -56,6 +56,9 @@ properties:
     description: |
       Phandle to the rifsc device to check access right.
 
+  power-domains:
+    maxItems: 1
+
 required:
   - compatible
   - reg
@@ -96,6 +99,9 @@ patternProperties:
           - const: MCLK
         minItems: 1
 
+      power-domains:
+        maxItems: 1
+
       dmas:
         maxItems: 1
 
-- 
2.39.2

